readme.md

```diff
@@ -57,8 +57,7 @@ You must have Go > 1.6 installed.
     make build
 
 And to run hoverfly:
-
-    ./cmd/hoverfly/hoverfly
+    $GOPATH/bin/hoverflyfb
```

Comment:
Thanks for spotting this.  Your changes are correct, barring it should be `$GOPATH/bin/hoverflyb` instead of `$GOPATH/bin/hoverflyfb`.